collectors/python.d.plugin/pandas/README.md

```diff
@@ -0,0 +1,51 @@
+<!--
+title: "Pandas"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/python.d.plugin/pandas/README.md
+-->
+
+# Pandas Netdata Collector
+
+A python collector using [pandas](https://pandas.pydata.org/) to pull data and do pandas based preprocessing before feeding to Netdata.
+
+## Requirements
+
+This collector depends on some Python (Python 3 only) packages that can usually be installed via `pip` or `pip3`.
+
+```bash
+$ sudo pip install pandas requests
+```
+
+## Configuration
+
+Below is an example configuration to query some csv data from [london Netdata demo server](http://london.my-netdata.io/#after=-420;before=0;=undefined;theme=slate;utc=Europe%2FLondon), do some data wrangling on it and save in format as expected by Netdata.
+
+```yaml
+# example showing a read_csv from a url and some light pandas data wrangling.
+# pull data in csv format from london demo server and then ratio of user cpus over system cpu averaged over last 60 seconds.
+example_csv:
+    name: "example_csv"
+    update_every: 2
+    chart_configs:
+      - chart_name: "london_system_cpu"
+        chart_title: "london_system_cpu"
+        chart_family: "london_system_cpu"
+        chart_context: "london_system_cpu"
+        chart_type: "line"
+        chart_units: "n"
```

Comment:
suggestions:
 - do not use the `chart_` prefix, I think it is already clear from the context (chart_configs).
 - provide a proper name, title, units, etc since you this is an example in the readme.

Reply:
done

Reply:
I think "ctx: pandas" (does not look like a metric name) and "units: n" are not good examples.

Reply:
Yeah tbh I found it hard to come up with even those. I might try work on a weather api example and use that if I can find a decent free api since that would be much more intuitive to a user.

Reply:
"family" and "context" really confuse me is the problem :)

Reply:
and i did just read 

https://learn.netdata.cloud/docs/dashboard/dimensions-contexts-families#context
and
https://learn.netdata.cloud/docs/dashboard/dimensions-contexts-families#family

but my brain just refuses to "get it" tbh

Reply:
but thanks - i actually was going to just ask you for the answer above.